### ISSUE TITLE
Show existing ad-hoc toolkits to Kits platform

### ISSUE THREAD

**Issue by swwalker** [DukeLearningInnovation/kits#470]
I am a current user of Toolkits creating my own adhoc groups.  I want to be able to access my existing adhoc group in the Kits application.  

**Questions**
Are the permission levels the same for course-related Kits, non course-related Kits, and user created Kits the same? - Yes, until a case is made for differing roles

Are the role names for course-related Kits, non course-related Kits and user-created Kits the same?  

What are the implications if they are not the same, 
- from a group manager perspective?
- from a merged kit perspective?

(related to #453 Define roles for user-created kits)

**Assumptions**
Adhoc toolkits that are no longer in use should not be migrated.
_Note: per Liz Wendland, there are currently no retention policies in place for removal of unused toolkits._

**Hypotheses**

**Tasks**

- [x] Complete an inventory of the current ad-hoc toolkits in Toolkits.  Include name, when created, owner, purpose, and tools included
- [ ] Assign all project category

**Comment by swwalker** [comment]
@joliet - the ad-hoc tookit inventory should also include the type (non-SIS course, collaboration) per the discussions around role names and combining kits.  See #507

**Comment by joliet** [comment]
Something is amiss here.  I can see adhoc Toolkits in Kits, but I can't seem to access the Add App menu item for any kit including course kits that I can confirm in Toolkits that I have instructor access.  Example: COMPSCI 101L - 001 Sp 20 kit.  When I click on Add App I see the attached error.  When I tested the adhoc kits showing in Kits now, I impersonated Todd Stabley and see his adhoc Toolkits in Kits.  When I click Add App on one of his kits (impersonating Todd still), I also see this error.
<img width="632" alt="Page_Doesnt_Exist" src="https://user-images.githubusercontent.com/7387760/79013098-ec876200-7b35-11ea-865b-c5d3400c1f05.png">
